plugin.yaml

```diff
@@ -1,16 +1,9 @@
 plugins:
-  # Name could be anything, this name is what appears on the beginning of operation
-  # mappings.
-  gcp_plugin:
-    # Could be 'central_deployment_agent' or 'host_agent'.
-    # If 'central_deployment_agent', this plugin will be executed on the
-    # deployment dedicated agent, other wise it will be executed on the host agent.
+  gcp:
     executor: central_deployment_agent
-
-    # URL to archive containing the plugin or name of directory containing
-    # the plugin if it is included in the the blueprint directory under the
-    # "plugins" directory.
     source: https://github.com/cloudify-cosmo/cloudify-gcp-plugin/archive/master.zip
```

Comment:
Source should be changed to 1.0.zip
The 1.0 tag will be auto generated by the wagon build process after merging this PR.